Canvas/Canvas/CanvasAppDelegate.swift

```diff
@@ -107,6 +107,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDelegate {
 
     func setupUITests() {
         Keychain.clearEntries()
+        UIView.setAnimationsEnabled(false)
```

Comment:
Are we sure we want to disable animations entirely? Or would it be better to just speed them up, like we do in StudentUITests?

Reply:
Ooo I will speed them up instead.